### DISCUSSION TITLE
Question regarding adding a Billing user to a Team to set up payments

### DISCUSSION THREAD

**Discussion by alexwaddell97** [vercel/community#3622] [Feature Question] [answered]
### Summary

I'm trying to pin down a process for setting up clients billing. The plan was to invite clients to a team and then have them set up the Pro subscription as a Billing role user and I'd be the owner/developer. Problem being this doesn't work this way and I'd have to upgrade to Pro as far as I'm aware before I'd be able to invite a Billing user at no cost. Is this how it works or am I missing something?

**Comment by amyegan** [comment, marked as answer]
Hey @alexwaddell97. Yes, the owner of the team would need to create the team because the Billing role has no ability to manage team and project settings. 

> The [Billing role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#billing-role) is specifically designed for those who need to manage the financial aspects of the team, such as handling billing settings and overseeing invoices. Individuals assigned to this role have **read-only access** to all projects within the team, providing a comprehensive view of the team's undertakings **without the ability to make modifications**. 

Project [owners](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#owner-role) each need their own seat on the team, but a team can have more than one owner.

**Reply by alexwaddell97** [reply]
Thanks for the response, is there maybe room in the future to allow the invitation of a billing seat to allow a third party to upgrade to the pro tier for a team?

Ideally trying to avoid the need for the person who we’d assign the billing role ie. The customer who’s projects we’d be managing for them to create accounts etc/create the team as an owner as a lot of them aren’t technically minded.
